tests/run/enum-Color.scala

```diff
@@ -0,0 +1,13 @@
+enum Color {
+  case Red
+  case Green
+  case Blue
+}
```

Comment:
Will

    enum Color { case Red, Green, Blue }

expand to the equivalent of the above?